GVRf/Framework/framework/src/main/java/org/gearvrf/GVRScript.java

```diff
@@ -104,6 +104,20 @@ public void onAfterInit() {
     }
 
     /**
+     * Called when the script is attached to a target using {@link GVRScriptManager#attachScript}.
+     */
+    @Override
+    public void onAttach(Object target) {
```

Comment:
Yes, the event can only be sent to something that can have a script attached so that makes sense. I will change it.